paasta_tools/cli/cmds/logs.py

```diff
@@ -1032,7 +1034,11 @@ def fake_context():
                 f"Running the equivalent of 'scribereader -e {scribe_env} {stream_name}'"
             )
             yield scribereader.get_stream_tailer(
-                stream_name, host, port, True, line_count
+                stream_name=stream_name,
+                tailing_host=host,
+                tailing_port=port,
+                lines=line_count,
+                superregion=cluster,
```

Comment:
This'll break in mesosstage. Maybe we need an extra SystemPaastaConfig parameter for this? Or the ability to read the superregion fact / `/nail/etc/superregion`

Reply:
We already have `cluster_map` in the  system paasta config which goes from paasta cluster -> "scribe env".

Sounds like we'll need a new cluster mapping to go from cluster=> superregion?

Reply:
After a bit of experimentation, it looks like the superregion arg is not necessary so I've removed. My initial testing returned a false negative because it didn't give me any logs, but that would've been the same whether or not the arg was there or not.